packages/fcl/src/current-user/exec-service/strategies/iframe-rpc.js

```diff
@@ -23,8 +23,31 @@ export function execIframeRPC(service, signable) {
         }
       },
 
-      onClose() {
-        reject(`Declined: Externally Halted`)
+      onResponse(e, {send, close}) {
```

Comment:
Trying to remove the concept of signable as future services probably aren't going to be about signing stuff.

```javascript
export function execIframeRPC(service, body, opts = {}) {
  const includeOlderJsonRpcCall = opts.includeOlderJsonRpcCall
  const id = uid()
  body.data = service.data
  
  frame(service, {
    onReady(_, {send}) {
      try {
        send({
          type: "FCL:FRAME:READY:RESPONSE",
          data: signable, // but signable -> data
          service: {
            params: service.params,
            data: service.data, // would be okay with including this
          },
        })
        
        if (includeOlderJsonRpcCall) {
          send({ jsonrpc: "2.0", id: id, method: "fcl:sign", params: [signable, service.params ]})
        }
      } catch (error) {
        throw error
      }
    }
  })
}
```

Reply:
data: signable, // but signable -> data

I believe this is what we are looking for?
data: body

Reply:
ahh yeah signable to body